### ISSUE TITLE
If updating index aliases just update those of the same domain

### ISSUE THREAD

**Issue by dr0i** [lobid/lodmill#689]
When indices are build in parallel the updating mechanism may switch an index which is not finished to the productive one.

Since hadoop is no more used (better: not the only way to generate an index) it happens that e.g. organisations' index is updated while also  resources is created, in parallel. Now when organisations is finished all indices are newly aliased with the newest index becoming the procutive one. When the newest resources index is niot finished , it regardless become the productove one. Such introducing this bug. 

In the past such parallel indexing was kind of impossible, because hadoop jobs run sequential, not in parallel (although it was possible, e.g. when resources were transformed and indexing began and transforming of organisations began and indexing of organisation while resource was indexing and organisations was outpacing the resources and switching index and voila).

Solution would be that e.g. if organisations are indexed only organisation indices are updated, not the rest.

See also #609.

**Comment by dr0i** [comment]
This issue is a duplicate of #609.

**Comment by dr0i** [comment]
Tested and deployed to production, closing.
